sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/Helpers/CosmosDBTestUtilities.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Management.Resources;
+using Azure.Management.Resources.Models;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.CosmosDB.Tests
+{
+    public static class CosmosDBTestUtilities
+    {
+        internal const string ResourceGroupPrefix = "DefaultCosmosDB";
```

Comment:
Use the recoding method like this to get a "unique" id: Recording.GenerateAssetName("Default-CosmosDB-");

Reply:
Here is the prefix to generate unique id, it is using the recording method in test file:
https://github.com/Azure/azure-sdk-for-net/blob/9e997cfdd7fc79b5355a0d7a4ebebac341bba931/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/OperationsTest.cs#L24